pkg/konfluxgen/imagerepository.template.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ImageRepository
+metadata:
+  annotations:
+    image-controller.appstudio.redhat.com/update-component-image: "true"
+  labels:
+    appstudio.redhat.com/application: {{ sanitize .ApplicationName }}
+    appstudio.redhat.com/component: {{ truncate ( sanitize ( print .ProjectDirectoryImageBuildStepConfiguration.To "-" .ReleaseBuildConfiguration.Metadata.Branch ) ) }}
+  name: imagerepository-for-{{ sanitize .ApplicationName }}-{{ .ProjectDirectoryImageBuildStepConfiguration.To }}
```

should we shorten it to `ir-` ? we might run out of characters (63 max) when combining app + component + long prefix 

```suggestion
  name: ir-{{ sanitize .ApplicationName }}-{{ .ProjectDirectoryImageBuildStepConfiguration.To }}
```

---

Good point about max length. I only took it from
```
k --context ocp-serverless@konflux get imagerepository           
NAME                                                                                    IMAGE                                                                                                                   VISIBILITY
...
imagerepository-for-serverless-operator-konflux-serverless-openshift-knative-operator   quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-konflux/serverless-openshift-knative-operator   public
imagerepository-for-serverless-operator-release-next-knative-eventing-controller        quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-release-next/knative-eventing-controller        public
```
which are created automatically when components are onboarded through the UI.

I think I would remove the prefix then entirely...